notebooks/enterprise-search/app-search-crawler-to-open-crawler-migration.ipynb

```diff
@@ -0,0 +1,383 @@
+    "es_client = Elasticsearch(\n",
+    "    \":\".join([ENDPOINT, PORT]),\n",
+    "    api_key=OUTPUT_API_KEY,\n",
+    ")\n",
+    "\n",
+    "es_client.info()[\"tagline\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14ed1150-a965-4739-9111-6dfc435ced4e",
+   "metadata": {},
+   "source": [
+    "#### **This is the final step! You have two options here:**\n",
+    "\n",
+    "- The \"Write to YAML\" cell will create _n_ number of YAML files, one for each Crawler you have.\n",
+    "- The \"Print to output\" cell will print each Crawler's configuration YAML in the Notebook, so you can copy-paste them into your Open Crawler YAML files manually.\n",
+    "\n",
+    "Feel free to run both! You can run Option 2 first to see the output before running Option 1 to save the configs into YAML files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a203432-d0a6-4929-a109-5fbf5d3f5e56",
+   "metadata": {},
+   "source": [
+    "#### Option 1: Write to YAML file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d6df1ff-1b47-45b8-a057-fc10cadf7dc5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dump each Crawler's configuration into its own YAML file\n",
+    "for crawler_config in inflight_configuration_data.values():\n",
+    "    base_dir = os.getcwd()\n",
+    "    file_name = (\n",
+    "        f\"{crawler_config['output_index']}-config.yml\"  # autogen a custom filename\n",
+    "    )\n",
+    "    output_path = os.path.join(base_dir, file_name)\n",
+    "\n",
+    "    if os.path.exists(base_dir):\n",
+    "        with open(output_path, \"w\") as file:\n",
+    "            yaml.safe_dump(crawler_config, file, sort_keys=False)\n",
+    "            print(f\" Wrote {file_name} to {output_path}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49cedb05-9d4e-4c09-87a8-e15fb326ce12",
+   "metadata": {},
+   "source": [
+    "#### Option 2: Print to output"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e99d6c91-914b-4b58-ad7d-3af2db8fff4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for crawler_config in inflight_configuration_data.values():\n",
+    "    yaml_out = yaml.safe_dump(crawler_config, sort_keys=False)\n",
+    "\n",
+    "    print(f\"YAML config => {crawler_config['output_index']}-config.yml\\n--------\")\n",
+    "    print(yaml_out)\n",
+    "    print(\n",
+    "        \"--------------------------------------------------------------------------------\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26b35570-f05d-4aa2-82fb-89ec37f84015",
+   "metadata": {},
+   "source": [
+    "### Next Steps\n",
+    "\n",
+    "Now that the YAML files have been generated, you can visit the Open Crawler GitHub repository to learn more about how to deploy Open Crawler: https://github.com/elastic/crawler#quickstart\n",
+    "\n",
```

Comment:
Read our search blog posts with [highlights from Open Crawler's promotion to beta](https://www.elastic.co/search-labs/blog/elastic-open-crawler-beta-release) or learn how to [use the Open Crawler in combination with Semantic Text to easily crawl web sites and make them semantically searchable](https://www.elastic.co/search-labs/blog/semantic-search-open-crawler).

Reply:
@mattnowzari How about adding a reference to existing blogs? ☝️